_site/index.html

```diff
@@ -593,7 +664,30 @@ <h3>T-Z</h3>
             </p>
           </div>
         </section>
-        
+  
+        <section>
+          <p class="word__title" id="pansexual">
+            Pansexual
+            <span class="word__speech">
+              adj
+            </span>
+          </p>
+          
+          <p class="word__definition">
+            someone who is sexually attracted to people regardless of their gender, or to people of any gender.
```

Comment:
For example, a non-binary person may have some gender between male and female, have multiple genders, no gender, a culturally-specific gender (e.g., two-spirit).